inspirehep/modules/theme/views.py

```diff
@@ -101,3 +117,32 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return make_response(jsonify(success=False), 400)
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return make_response(jsonify(success=False), 403)
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return make_response(jsonify(success=False), 403)
+
+    content = '''Feedback:
+{feedback}'''.format(feedback=feedback)
+    message = Message(sender=current_app.config['CFG_SITE_SUPPORT_EMAIL'],
+        recipients=[current_app.config['INSPIRELABS_FEEDBACK_EMAIL']],
+        subject='INSPIRE Labs Feedback',
+        body=content,
+        reply_to=replytoaddr)
+
+    send_email(message)
```

Comment:
...which is a known bug, already found by @eamonnmag, and solved [here](https://github.com/jirikuncar/invenio-accounts/commit/0a3caef5c1ae669074287fcbefc62c932e4b5d82) for `invenio-accounts`. I will submit a PR to `invenio-mail` with a similar fix.